luigi_pipeline/seqr_sv_loading.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import sys
+
+import luigi
+import hail as hl
+
+from lib.model.sv_mt_schema import SeqrSVVariantSchema, SeqrSVGenotypesSchema, SeqrSVVariantsAndGenotypesSchema
+from luigi_pipeline.seqr_loading_optimized import SeqrVCFToVariantMTTask, BaseVCFToGenotypesMTTask, BaseMTToESOptimizedTask
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode
+
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrSVVariantMTTask(SeqrVCFToVariantMTTask):
+    # Overrided inherited required params.
+    reference_ht_path = ""
+    clinvar_ht_path = ""
+    sample_type = "WGS"
+    genome_version = "38"
+    dont_validate = True
+    dataset_type = "SV"
+
+    gencode_release = luigi.IntParameter(default=43)
+    gencode_path = luigi.OptionalParameter(default="", description="Path for downloaded gencode data")
+    RUN_VEP = False
+    SCHEMA_CLASS = SeqrSVVariantSchema
+
+    # NB: electing not to override import_vcf here eventhough the inherited args are slightly different
+    # than from the old pipeline.
+
+    def get_schema_class_kwargs(self):
+        return {
+            "gene_id_mapping" : hl.literal(load_gencode(self.gencode_release, self.gencode_path))
```

Comment:
Opted to just use the existing `load_gencode` function for now... though I imagine we'll want to port it to something that looks like what we have in `download_and_create_reference_datas`.

Reply:
I agree that downloading all of genocode every single time we run the pipeline is overkill just to get a gene symbol to id mapping. It probably makes the *most* sense to ask the team that delivers the file to do so with ensemble IDs, which is what we asked them to do for gCNV data. But in the interim this is fine